### ISSUE TITLE
SSH usage

### ISSUE THREAD

**Issue by PliniLeonardo** [AlbertoPdRF/root-file-viewer#31]
Dear Alberto,

This is a wonderful project: I love it! 
May I ask you if it possible to see the plots even with a ssh connection to a server that has root-file-viewer extension? 

Best,
Leonardo

**Comment by AlbertoPdRF** [comment]
Thanks for the kind words @PliniLeonardo!

In principle there should be no problem using the extension in conjunction with the [Remote - SSH extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh). Please let me know if you have any problem!

**Comment by PliniLeonardo** [comment]
Yes, it seems like it opens the file with the extension, but then the TTrees are not visible. However, this statement is valid only when using an ssh connection since I am able to look at the plots when using the server.
